client/packages/host/src/api/hooks/utils/useHostApi.ts

```diff
@@ -10,6 +10,7 @@ export const useHostApi = () => {
       [...keys.base(), 'initialisationStatus'] as const,
     syncStatus: () => [...keys.base(), 'syncStatus'] as const,
     initialiseSite: () => [...keys.base(), 'initialiseSite'] as const,
+    updateSyncSettings: () => [...keys.base(), 'updateSyncSettings'] as const,
```

Comment:
not required

Reply:
🙏 70a49f5ee7ffbd237340e0ed8fbac838c5d9afc8